15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md

```diff
@@ -19,6 +19,19 @@ Use the [general upgrade guide](../) to complete the upgrade of your project.
 
 <details>
 
+<summary>Umbraco 15</summary>
+
+* **Snapshots are removed**
+
+Snapshots has been removed, this means that any code that used `IPublishedSnapshot`, and by extension `IPublishedSnapshotAccessor`, must be updated. Inject `IPublishedContentCache` or `IPublishedMediaCache` and use those directly instead.
+
+* **Modelsbuilder models needs to be rebuilt**
+
+Models generated by ModelsBuilder used the `IPublishedSnapshot` interface, which has been removed. This means that the models needs to be rebuilt, the approach to this will differ on the models mode chosen:
```

Comment:
```suggestion
Models generated by ModelsBuilder used the `IPublishedSnapshot` interface, which has been removed. This means that the models need to be rebuilt. The approach to this will differ depending on the mode chosen:
```